src/chrome/content/rules/infoseek.xml

```diff
@@ -0,0 +1,42 @@
+<!--
+
+	Nonfunctional domains:
+
+		- infoseek.co.jp subdomains:
+
+			- transfer *
+			- translation *
+			- point ²
+			- schedule **
+			- special **
+
+	* Times out
+	² Redirects to http
+	** Refused
+	*** 404, valid cert
+
+	Problematic subdomains:
+
+		- flash.api.earthquake	(invalid cert)
+		- mero	(invalid cert)
+
+-->
+<ruleset name="Infoseek" platform="mixedcontent">
+
+	<target host="image.infoseek.rakuten.co.jp" />
```

Comment:
Is rakuten.co.jp related with infoseek.co.jp?
Please add a comment why this needs to be in this ruleset or put it in a another file.